### PR TITLE
Australia Rules: Content from the Melbourne Instructor Training

### PR DESCRIPTION
**Pull request by BillMills**
Hey all,

Here's my new content generated for the instructor training I led for ResBaz at the University of Melbourne, February 2015.

## Summary:

 - Added a new chapter on feedback. Discusses outside-in, rubrics, and techniques for giving useful feedback.
 - Added new chapter on compelling teaching & inclusivity.
 - Formatting fixes on old RID material
 - New exercises throughout
 - Added a proposed schedule that reflects what we did in Melbourne.

## To-Do:
 - Didn't recompile html as is the new policy, but note it is way out of whack now due to new sections added.
 - `inclusivity`, `motivation`, and the awesome new `diversity` chapters are all coming at the same content; we need to combine our powers here. The content in `inclusivity` is how I think about this material; I'd like the authors of the other two to be in charge of deciding what to keep of that, what to dump, and how to merge all three efforts.

On a personal note, big thanks to @gvwilson for mentoring me and giving me the chance to teach this course; I'd be delighted to do it again soon. Also thanks to @DamienIrving and the ResBaz team for having me over so we could try these ideas out - I think it was a big success.

Also, cc @ethanwhite, since you were asking after this content :)

### CODE REVIEW

**Comment by wking** [comment]
On Sat, Mar 21, 2015 at 11:13:20AM -0700, Bill Mills wrote:
>     M 02-concepts.md (24)

Heh, an 18x increase in expected time with *less* content :p.

>     A 04-inclusivity.md (101)

This is great :) (although I'm not familiar enough with the rest of
the content to know how it fits in).  Minor copy-edits:

* Drop the comma in “What we haven't addressed yet, are the confounding
  variables and psychological effects…”.
* “subject wide” → “subject-wide”.
* I'd use http://doi.org/… links, since that's probably more stable
  than linking directly into a publisher's site.
* “git's potency” → “Git's potency”.  And maybe “the value in” → “the
  value of” at the end of that sentence.

My only pedagogical concern here is that, while you present a bunch of
useful issues to think about, the topic as a whole seems a bit out of
focus.  I'd lay out the research at the beginning (shifting the How
Learning Works reference up to sit near the Student Self-Perception
section).  Then break the rest of the section down into elaborations
on the research-based components: a section about self-concept,
another about self-efficacy (which I'd break down further into
expectations of success and a supportive environment), another about
value.  I'd start off with value, since the value pitch ties into the
topic definition.  Then talk about addressing self-concept issues that
students may bring to a workshop, since we can only mitigate issues
there.  Finally, have the bulk of the topic in a sections on
self-efficacy and constructing and environment to keep students
motivated without falling into any of the common traps.  I think
you'll have better luck sticking the big ideas if you keep coming back
to and reinforcing the three-component framework with every example.
The finer details are good, but you're unlikely to stick *all* of them
and you want the takeaway message to degrade gracefully in the face of
lossy communication ;).

File: 03-feedback.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+subtitle: Feedback Strategy
+minutes: 180
+---
+> ## Learning Objectives {.objectives}
+>
+> * FIXME
+
+##Introduction
+
+In our discussion of our three-stage model of expertise, we concluded that most scientists don't need to be expert software developers; it's enough for them to be able to solve a recurring cast of coding problems in more-or-less familiar circumstances, which is the halmark of a competent practitioner. Teaching, however, is in some sense always a novel problem. Students will always come up with new ways to misunderstand things, and so the flexibility afforded by mastery in teaching is always valuable. How does one go about becoming a master instructor?
```
Comment (wking) [inline 03-feedback.md]: “halmark” → “halmark”.  Or maybe just “mark”?

Comment (wking) [inline 03-feedback.md]: Err → “hallmark” ;).

File: 03-feedback.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The popular ten thousand hours number is certainly a myth. Many people have driven for more than ten thousand hours, and are little closer to mastery than after their driver's exam. Something more than mere repetition is required.
+
+Positing additional linkages between ideas is necessary but also not sufficient; there needs to be some mechanism to scrutinize and validate those new links and the strategies that come out of them.
+
+Musicians, athletes, actors and politicians have been doing it for years - the key is simple feedback. By having a teacher or even a colleague observe and give constructive criticism that is specific and actionable, we can iterate on our practice and get that invaluable second perspective on whether what we're doing is working, and where it's falling short. In time, with enough criticism, we can begin to usefully criticize our own practice, and this self-reflective criticism greatly accelerates the process of gaining mastery. But to begin with, getting feedback from our colleagues starts the process.
```
Comment (wking) [inline 03-feedback.md]: I agree that external criticism (especially from more experienced folks) is extremely useful.  I think this paragraph is phrased a bit too strongly though.  Bootstrapping based solely on practice and self-assessment is possible, it's just going to take you longer and you'll go down more dead ends.  I can try to come up with some specific rewording suggestions, but I'm sure you can as well (if my argument is convincing ;).  Let me know if you do want rewording suggestions.

File: 03-feedback.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+##Anatomy of Feedback
+
+The practice of giving feedback can be incredibly powerful in its ability to expose things we couldn't perceive in ourselves in the intensity of the stage, but it can also be a very fraught process. Criticism can challenge our self image, and chafe on our insecurities and doubts; receiving criticism can feel like a place of powerlessness. If we are to successfully critique each other, we must find a way to speak our minds in a way each other can hear - how do we criticize each other in a way designed to empower and not to tear down? Giving good criticism consists of three cardinal rules:
```
Comment (wking) [inline 03-feedback.md]: Again, this seems a bit strong.  Is there research backing up these three rules?  I imagine there is a good deal of research on criticism, but I'm not familiar with any of it.  In the absense of a more evidence-based argument, I'd prefer wording like: “Here are a few techniques that may help you give constructive criticism:”

File: 03-feedback.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+ - **Be polite.** While it seems trite in writing, social gaffes are never far away when giving criticism. A critical conversation often can't help but exceed the usual boundaries of frankness and honesty; best then to be especially mindful of framing comments in a positive light, describing opportunities to improve, and not going further than we must.
+
+[Exercise 1: Basic Feedback](http://mozillascience.github.io/instructorTraining/feedback/feedback.html)
+
+##Structured Feedback
+
+Trying to articulate exactly where your fellow instructor should turn her attention to in order to improve can be a nebulous and difficult task in practice. Things can go much more smoothly with a bit of structure to the process.
```
Comment (wking) [inline 03-feedback.md]: “turn her attention to in order” → “turn her attention in order”.  No big deal, but I think it reads more clearly that way.

Comment (wking) [inline 03-feedback.md]: And I'd drop “in practice”.

File: 03-feedback.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+###Outside-In
+
+'Outside-In' feedback is a technique for loosely structuring feedback popularized in theater. It groups feedback into three main categories: physical, presentation, and content.
+
+ - **Physical feedback** focuses on the raw physical actions of the presenter; are they doing anything physically that distracts from or interferes with their lesson? Common examples are facing the board while speaking, nervous fidgeting like fiddling with hair or juggling things in their pockets, and extreme pacing.
+
+ - **Presentation feedback** focuses on the manner of delivery of the lesson; is the presenter effectively commanding the attention of the room? Common mistakes are mumbling and speaking too quietly, growing impatient with mistakes or questions, and speaking too fast to follow.
+
+ - **Content feedback** focuses on the actual material and its layout in the lesson; is the lesson planned and constructed as well as possible? Common mistakes include skipping 'obvious' steps, making forward references to things that haven't been explained yet, and dwelling on minuta when the basics haven't yet been clarified.
```
Comment (wking) [inline 03-feedback.md]: “minuta” → “minutia”.

File: 02-concepts.md [hunk 6]
```diff
@@ -148,27 +148,7 @@ The novel problems that scientists encounter will almost entirely be in their fi
...
-
-Musicians, athletes, actors and politicians have been doing it for years -
-the key is simple feedback. By having a teacher or even a colleague observe and give constructive criticism that is specific and actionable,
-we can iterate on our practice and get that invaluable second perspective on whether what we're doing is working,
-and where it's falling short. In time, with enough criticism, we can begin to usefully criticise our own practice,
-and this self-reflective criticism greatly accelerates the process of gaining mastery.
-But to begin with, getting feedback from our colleagues starts the process.
-
-[Exercise 3: Feedback Pt. 1](http://mozillascience.github.io/instructorTraining/knowledgeAndExpertise/feedback.html)
+[Exercise 3: Targeting Novices](http://mozillascience.github.io/instructorTraining/knowledgeAndExpertise/targetingNovices.html)
```
Comment (kynan) [inline 02-concepts.md]: Are these links still up to date? Could they be relative?

Comment (BillMills) [inline 02-concepts.md]: Yep, the links are all still there, and aren't going to break any time soon given the grace of GitHub :) They link out to teaching kits, which was okay in the past - would we like markdownified challenges inserted into the text?

**Comment by BillMills** [comment]
@wking thanks for all your comments here! All your wording suggestions just got merged, thanks. Re: structure from your original post, I think that's a strong case - but this PR is exactly what I did in Melbourne, which was a big success. I would strongly encourage someone to teach it your way next time - and PR your changes in if they go well in practice (which I think they most likely will).

**Comment by wking** [comment]
On Mon, Apr 06, 2015 at 11:31:13AM -0700, Bill Mills wrote:
> Re: structure from your original post, I think that's a strong case
> - but this PR is exactly what I did in Melbourne, which was a big
> success. I would strongly encourage someone to teach it your way
> next time - and PR your changes in if they go well in practice
> (which I think they most likely will).

+1 to the evidence-based approach.  If this lands and I get some time,
I'll PR the restructured content so future instructors interested in
testing it will (a) be able to find it and (b) not have to do too much
work before they can test it.

File: 03-feedback.md (hunk 2 above)
Comment (wking) [inline 03-feedback.md]: On Mon, Apr 06, 2015 at 11:20:43AM -0700, Bill Mills wrote:
> I think the idea is that that useful self-reflection is actually a
> high-level skill, and it takes time to get there. But, I'd love to
> see a re-write that accommodates both points while still leading
> smoothly into talking about (external) feedback.

How about something like:

  Mastering a skill is a process of practice and feedback.  If you
  don't test yourself or judge what's working and what's not working,
  you won't know what to improve.  It's possible to critique your own
  performance, but it takes time to collect the experience necessary
  to make informed decisions in a new arena.  It's much more
  efficient, especially when you're just getting started, to get
  feedback from a more experienced coach or colleague.

**Pull request closed without merging**